Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+// vim: ft=groovy
+
+properties([
+    parameters([
+        string(name: 'TARGET', defaultValue: '', description: '')
+    ])
+])
+
+node('arm64') {
+    deleteDir()
+    checkout scm
```

Comment:
I believe we should check that the `$TARGET` is set up properly before we do anything else.